packages/gatsby/content/getting-started/install.md

```diff
@@ -8,19 +8,21 @@ order: 2
 
 ## Install Corepack
 
-The preferred way to manage Yarn is through [Corepack](https://nodejs.org/dist/latest/docs/api/corepack.html), a new binary shipped with all Node.js releases starting from 16.10. It acts as an intermediary between you and Yarn, and lets you use different package manager versions across multiple projects without having to check-in the Yarn binary anymore.
+The preferred way to manage Yarn is through [Corepack](https://nodejs.org/dist/latest/docs/api/corepack.html), a new binary shipped with all Node.js releases starting from 16.9 and in v14 releases starting from 14.19. It acts as an intermediary between you and Yarn, and lets you use different package manager versions across multiple projects without having to check-in the Yarn binary anymore.
 
-### Node.js >=16.10
+> **Note:** The Corepack version shipped with 16.9.0 and 16.9.1 contains a bug preventing the execution of `yarn init -2`, so instead of enabling this version, we continue as if it did not exist.
 
-Corepack is included by default with all Node.js installs, but is currently opt-in. To enable it, run the following command:
+### Node.js >=16.10, >= 14.19
+
+Corepack is included by default with all Node.js installs, but is currently opt-in. To enable it, run the following command (may need root):
 
 ```bash
 corepack enable
 ```
 
-### Node.js <16.10
+### Node.js <16.10, <14.9
 
-Corepack isn't included with Node.js in versions before the 16.10; to address that, run:
+There is no usable version of Corepack included with Node.js in versions before 16.10 and 14.19; to address that, run (may need root):
```

Comment:
```suggestion
For versions of Node.js that doesn't include Corepack by default you can install it manually by running the following command
```

Reply:
> my idea was to prevent users from just trying to use the 16.9-version, when they find out about it

This is why the documentation currently references 16.10. Imo the current wording is fine, we only received this one report that it might be confusing; I'd probably tend to keep it unless we see a different wording that's an unambiguous clear improvement 🤷‍♀️

Reply:
I started believing you overlooked 16.9, when I realized 14.19 also included Corepack. So we are already at 2.

How about
```
For releases of Node.js that don't include a suitable version of Corepack by default you can install it manually by running the following command
```
Fixes `doesn't` used on a plural and hints, that there are unsuitable versions; details are already in the note-box in the previous paragraph.